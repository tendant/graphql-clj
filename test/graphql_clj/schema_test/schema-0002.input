type Person {
name(id: ID id2: ID): String
age: Int
picture: String
}