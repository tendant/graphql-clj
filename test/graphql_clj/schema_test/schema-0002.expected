{:objects
 {:Person
  {:fields
   {:name {:args {"id" {:type ID}, "id2" {:type ID}}, :type String},
    :age {:type Int},
    :picture {:type String}}}}}
